Tasks/choco/Choco.ps1

```diff
@@ -62,30 +66,46 @@ function Install-Package
         [string] $IgnoreChecksums
     )
 
-    $expression = "$ChocoExePath install -y -f --acceptlicense --no-progress --stoponfirstfailure $Package --version $Version"
+    $expression = "$ChocoExePath install $Package"
+    
+    if ($Version){
+        $expression = "$expression --version $Version"
+    }
+
+    $expression = "$expression -y -f --acceptlicense --no-progress --stoponfirstfailure"
     
     if ($IgnoreChecksums -eq "true") {
         $expression = "$expression --ignorechecksums"
     }
 
-    Execute -Expression $expression
+    Set-ExecutionPolicy Bypass -Scope Process -Force
+    $packageScriptPath = [System.IO.Path]::GetTempFileName() + ".ps1"
+    Set-Content -Value $expression -Path $packageScriptPath
+        
+    Execute -File $packageScriptPath
+
+    # Check if the package installation failed
+    if ($LASTEXITCODE -ne 0) {
```

Comment:
@DavidObando The LASTEXITCODE check isn't working here. Is it in the wrong place? Or is there a different check I could run? (it is returning 1 even through the installation was successful)